libcore/src/mpi/LCGrid.h

```diff
@@ -59,7 +59,7 @@ class LCGrid
     double _gridXmin, _gridXmax, _gridYmin, _gridYmax;
     /// for convenience
     /// will be delete in next versions
-    Pedestrian ** _localPedsCopy;
+    std::map<int, Pedestrian *> _localPedsCopy;
```

Comment:
I think this name is very much misleading, because the local copy is actually not a copy. We should rename this.